cvat-core/src/annotations.js

```diff
@@ -284,6 +284,22 @@
         return result;
     }
 
+    async function importDataset(instance, format, file, updateStatusCallback) {
+        if (!(format instanceof String || typeof format === 'string')) {
+            throw new ArgumentError('Format must be a string');
+        }
+        if (!(instance instanceof Project)) {
+            throw new ArgumentError('Instance should ne a Project isntance');
+        }
+        if (!(typeof updateStatusCallback === 'function' || updateStatusCallback === null)) {
+            throw new ArgumentError('Callback should ne a function or null');
+        }
+        if (!(file instanceof File && file.name.split('.').reverse()[0])) {
+            throw new ArgumentError('File should be file instance with ZIP extension');
```

Comment:
Why don't we support other archive extensions?
I don't think it is a big deal

Reply:
Can we check file.type? (as in cvat-ui) to be equal one of: 'application/zip', 'application/x-zip-compressed'